book/writing-programs/patched-crates.md

```diff
@@ -46,44 +49,97 @@ sha3 = { git = "https://github.com/sp1-patches/RustCrypto-hashes", package = "sh
 
 An example of using patched crates is available in our [Tendermint Example](https://github.com/succinctlabs/sp1/blob/main/examples/tendermint/program/Cargo.toml#L22-L25).
 
+## Ed25519 Acceleration
+
+To accelerate Ed25519 operations, you'll need to patch crates depending on if you're using `ed25519-consensus` or `ed25519-dalek`.
+
+Generally, `ed25519-consensus` has better performance than `ed25519-dalek` by a factor of 2.
+### Patches
+
+Apply the following patches based on your original crate usage:
```

Comment:
nit: prefer "based on which crate you're using"

Reply:
Fixed